bin/run.sh

```diff
@@ -12,6 +13,10 @@
 # First runs the tests with bats and saves the TAP output,
 # then parses that TAP file to produce the JSON file.
 
+# the version of the test-runner interface:
+# https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md
+INTERFACE_VERSION=2
```

Comment:
… but this is a static entry, should we link to that file with the SHA link instead of the dynamic version (with `main`)?